src/components/ScoreBoard/index.tsx

```diff
@@ -30,12 +30,12 @@ function getRoundScore(
 ) {
     let score = null;
 
-    const { bet, taken } = scores[roundNumber]?.[playerId] || {};
+    const { bet, taken = 0 } = scores[roundNumber]?.[playerId] || {};
```

Comment:
not sure if it ever is, but I have taken typed as number or null so setting default wouldn't work

Reply:
k, will also cast then

Reply:
now bets other peoples bets are visible to players who haven't bet yet

Reply:
also score is being calculated before trick has been played messing with the overall scores